integration-tests/gatsby-source-wordpress/gatsby-config.js

```diff
@@ -15,17 +15,42 @@ const mediaItemTypeSettings = {
   },
 }
 
+const sharedTypeSettings = {
+  Settings: {
+    excludeFieldNames: [`generalSettingsEmail`],
+  },
+  GeneralSettings: {
+    excludeFieldNames: [`email`],
+  },
+  WPGatsby: {
+    exclude: true,
+  },
+  PostTypeSEO: {
+    excludeFieldNames: [`twitterImage`],
+  },
+  BlockAttributesObject: {
+    exclude: true,
+  },
+  Post: { excludeFieldNames: [`pinged`] },
+}
```

Comment:
On latest versions of plugins these fields cause GraphQL errors. This is normally fine because errors that return data are silenced in source-wordpress. I turned that off for tests (below) so we can see all errors. I also turned on the debugging option to panic on errors - this is so we can see very quickly when a gql error is happening and fail the test

Reply:
See the lines below 👇